atmo/settings.py

```diff
@@ -114,6 +120,21 @@
     )
 }
 
+REDIS_URL = config(
+    'REDIS_URL',
+    cast=parse.urlparse
+)
+
+RQ_QUEUES = {
+    'default': {
+        'HOST': REDIS_URL.hostname,
+        'PORT': REDIS_URL.port,
+        'DB': 0,
+        'PASSWORD': REDIS_URL.password,
+        'DEFAULT_TIMEOUT': 360,
```

Comment:
I don't mind changing it, we can still tweak it later when we have a better idea of the typical task duration